### PR TITLE
feat(atomic): add outline color & ring color

### PR DESCRIPTION
**Pull request by ThibodeauJF**
https://coveord.atlassian.net/browse/KIT-880

### CODE REVIEW

File: packages/atomic/src/components/facets-v1/facet-search/facet-search-input.tsx [hunk 1]
```diff
@@ -25,15 +25,14 @@ export const FacetSearchInput: FunctionalComponent<FacetSearchInputProps> = (
...
-          class="w-full h-full border border-neutral bg-background rounded px-9 placeholder-neutral-dark text-sm hover:border-primary-light focus:border-primary focus:outline-none group"
+          class="input-primary w-full h-full px-9 placeholder-neutral-dark text-sm group"
           type="text"
           placeholder={search}
           aria-label={facetSearch}
           value={props.query}
           onInput={(e) => props.onChange((e.target as HTMLInputElement).value)}
           ref={(ref) => (inputRef = ref)}
         />
-        <div class="input-ring invisible ring-2 w-full h-full absolute inset-0 pointer-events-none rounded opacity-40 ring-primary"></div>
```
Comment (ThibodeauJF) [inline packages/atomic/src/components/facets-v1/facet-search/facet-search-input.tsx:36]: This was the previous "hack" to put a transparent ring around the search input using the primary color. IMO it is not super clean and is a pain to reuse the hack elsewhere, opted for a new color instead.

File: packages/atomic/tailwind.config.js [hunk 2]
```diff
@@ -55,5 +57,16 @@ module.exports = {
       backgroundColor: ['group-focus'],
     },
   },
-  plugins: [],
+  plugins: [
+    plugin(function ({addUtilities, theme}) {
+      addUtilities(
+        {
+          '.outline-color-primary-light': {
```
Comment (ThibodeauJF) [inline packages/atomic/tailwind.config.js:64]: Only one color for the outline is necessary and should be reused on elements displaying an outline.

File: packages/atomic/src/themes/coveo.css [hunk 3]
```diff
@@ -4,6 +4,7 @@
   --atomic-primary-light: #399ffe;
   --atomic-primary-dark: #1a50ad;
   --atomic-on-primary: #ffffff;
+  --atomic-ring-primary: rgba(19, 114, 236, 0.5);
```
Comment (ThibodeauJF) [inline packages/atomic/src/themes/coveo.css:7]: I've taken the primary color & set an opacity 50% using rgba, unfortunately we cannot change the opacity of a CSS variable like it was a normal property, hence the new color

File: packages/atomic/src/global/global.pcss [hunk 4]
```diff
@@ -22,4 +22,16 @@
   .link {
     @apply text-primary hover:text-primary-light hover:underline focus:text-primary-light focus:underline;
   }
+
+  .input-primary {
```
Comment (ThibodeauJF) [inline packages/atomic/src/global/global.pcss:26]: Those classes are very useful for reusibility!

**Review by nbuissoncoveo — APPROVED** [review]

File: packages/atomic/tailwind.config.js (hunk 2 above)
Comment (samisayegh) [reply packages/atomic/tailwind.config.js:64]: If there is only one, could the name be shortened to `.outline-color`?

**Review by samisayegh — APPROVED** [review]

**Pull request merged**